packages/yoga/src/Theme/themes/components/dropdown.js

```diff
@@ -0,0 +1,141 @@
+const dropdown = ({
+  colors,
+  fontSizes,
+  fontWeights,
+  lineHeights,
+  radii,
+  spacing,
+  borders,
+}) => ({
+  width: 320,
+  selector: {
+    background: colors.white,
+    padding: {
+      top: spacing.medium,
+      right: spacing.medium,
+      bottom: spacing.medium,
+      left: spacing.medium,
+    },
+    border: {
+      radius: radii.rounded,
+      color: colors.gray[4],
+    },
+  },
+  arrow: {
+    fill: colors.gray[7],
+  },
+  input: {
+    font: {
+      color: colors.gray[7],
+      size: fontSizes.small,
+      lineHeight: lineHeights.small,
+    },
+  },
+  button: {
+    paddingRight: spacing.medium,
```

Comment:
We usually set the padding prop as an object which contains all directions.

```suggestion
    padding: { 
       right: spacing.medium,
    },
```